### PR TITLE
NetworkGetPresenceInviteSessionId and NetworkGetPresenceInviteInviter Update

### PR DESCRIPTION
**Pull request by PatchByte**
NetworkGetPresenceInviteInviter: image
NetworkGetPresenceInviteSessionId: image

I changed the return value and the parameters!

### CODE REVIEW

File: NETWORK/NetworkGetPresenceInviteInviter.md [hunk 1]
```diff
@@ -6,11 +6,13 @@ aliases: ["0x4962CC4AA2F345B7"]
 
 ```c
 // 0x4962CC4AA2F345B7 0x19EC65D9
-Any NETWORK_GET_PRESENCE_INVITE_INVITER(Any p0);
+char* NETWORK_GET_PRESENCE_INVITE_INVITER(int inviteIndex);
+```
+```
```
Comment (stannum-cfx) [inline NETWORK/NetworkGetPresenceInviteInviter.md:11]: Please don't make descriptions in code blocks as outlined [here](https://github.com/citizenfx/natives/blob/master/.github/CONTRIBUTING.md).

File: NETWORK/NetworkGetPresenceInviteSessionId.md [hunk 2]
```diff
@@ -6,11 +6,13 @@ aliases: ["0x26E1CD96B0903D60"]
 
 ```c
 // 0x26E1CD96B0903D60 0xC5E0C989
-Any NETWORK_GET_PRESENCE_INVITE_SESSION_ID(Any p0);
+Hash NETWORK_GET_PRESENCE_INVITE_SESSION_ID(int inviteIndex);
+```
+```
+The Native returns a hash of the session id as string from the specific invite index!
```
Comment (stannum-cfx) [inline NETWORK/NetworkGetPresenceInviteSessionId.md:12]: Same as above, please don't make descriptions in code blocks (outlined [here](https://github.com/citizenfx/natives/blob/master/.github/CONTRIBUTING.md)).

**Review by stannum-cfx — CHANGES_REQUESTED** [review]

**Comment by PatchByte** [comment]
oh ok sorry!
i am going to correct it!

**Comment by PatchByte** [comment]
ok i corrected it 👍

File: NETWORK/NetworkGetPresenceInviteInviter.md [hunk 3]
```diff
@@ -6,11 +6,11 @@ aliases: ["0x4962CC4AA2F345B7"]
 
 ```c
 // 0x4962CC4AA2F345B7 0x19EC65D9
-Any NETWORK_GET_PRESENCE_INVITE_INVITER(Any p0);
-```
-
+char* NETWORK_GET_PRESENCE_INVITE_INVITER(int inviteIndex);
 
+Returns the Name of the inviter of the specific selected Invite
```
Comment (stannum-cfx) [inline NETWORK/NetworkGetPresenceInviteInviter.md:11]: This should be moved one line down ;)

File: NETWORK/NetworkGetPresenceInviteSessionId.md [hunk 4]
```diff
@@ -6,11 +6,12 @@ aliases: ["0x26E1CD96B0903D60"]
 
 ```c
 // 0x26E1CD96B0903D60 0xC5E0C989
-Any NETWORK_GET_PRESENCE_INVITE_SESSION_ID(Any p0);
-```
+Hash NETWORK_GET_PRESENCE_INVITE_SESSION_ID(int inviteIndex);
 
+The Native returns a hash of the session id as string from the specific invite index!
+```
```
Comment (stannum-cfx) [inline NETWORK/NetworkGetPresenceInviteSessionId.md:12]: Code block should end just below this part:
```
Hash NETWORK_GET_PRESENCE_INVITE_SESSION_ID(int inviteIndex);
```

**Review by stannum-cfx — CHANGES_REQUESTED** [review]
2 small changes remain as commented above.

**Pull request merged**